scripts/helm/install.sh

```
@@ -53,8 +53,8 @@ Options:
   --dep-update                   Run helm dependency update.
   --fail-if-installed            Fail with a status code 1 if the helm release '$RELEASE_NAME' already exists in the $K8S_NAMESPACE namespace.
   --hosted-chart                 Install a hosted chart instead of the local chart.
-  --version   <version>          Set the version/version-range for the chart. Works only when used with the '--hosted' option.
-  --registry  <registry-url>     Set the registry URL for the hosted chart. Works only when used with the '--hosted' option. (Default: $DEFAULT_REGISTRY)
+  --version   <version>          Set the version/version-range for the chart. Works only when used with the '--hosted-chart' option.
```

Shall we throw an error if used without hosted chart?

---

Sure, that'd be nicer.